pincer/objects/guild/guild.py

```diff
@@ -444,33 +457,59 @@ async def modify_member(
 
     async def modify_member(self, _id: int, **kwargs) -> GuildMember:
         data = await self._http.patch(
-            f"guilds/{self.id}/members/{_id}", data=kwargs
+            f"guilds/{self.id}/members/{_id}",
+            data=kwargs
         )
         return GuildMember.from_dict(construct_client_dict(self._client, data))
 
+    async def ban(self, member_id: int, **kwargs):
+        """|coro|
+        Bans a guild member.
+        Parameters
+        ----------
+        member_id : :class: int
```

Comment:
I know this wasn't you but change this to 
```
member_id : :class:`int`
```

Reply:
This in fact was me, will fix in the evening